utilities/cassandra/test_utils.cc

```diff
@@ -65,5 +65,38 @@ int64_t ToMicroSeconds(int64_t seconds) {
 int32_t ToSeconds(int64_t microseconds) {
   return (int32_t)(microseconds / (int64_t)1000000);
 }
+
+int fromHex(char c) {
```

Comment:
http://en.cppreference.com/w/cpp/io/manip/hex  seems also do the work, but I'm not sure the performance impact.